src/nih_wayfinding/app/scripts/routing/directions-service.js

```diff
@@ -35,6 +37,38 @@
                 return RoutingResponseStub;
             }, 100, false);
         }
+
+        function getFlagIconName(flagType) {
+            // TODO: Write tests once actual icons exist
+            switch (flagType) {
+                case 'bench':
+                    return 'glyphicon-flash';
+                case 'hazard':
+                    return 'glyphicon-warning-sign';
+                case 'bathroom':
+                    return 'glyphicon-trash';
+                default:
+                    return 'glyphicon-info-sign';
+            }
+        }
+
+        function getTurnIconName(turnType) {
+            // TODO: Write tests once actual icons exist
+            switch (turnType) {
+                case 'straight':
+                    return 'glyphicon-arrow-up';
+                case 'left':
+                case 'slightleft':
```

Comment:
Sure, I'll add a note. Although, these are temporary until we get actual icons to display. Once that happens, I expect the slight case to be different.